controller/src/main/java/io/pravega/controller/task/Stream/StreamTransactionMetadataTasks.java

```diff
@@ -114,7 +128,7 @@ public Void initializeStreamWriters(final ClientFactory clientFactory,
 
     @VisibleForTesting
     public Void initializeStreamWriters(final String commitStreamName, final EventStreamWriter<CommitEvent> commitWriter,
-                                 final String abortStreamName, final EventStreamWriter<AbortEvent> abortWriter) {
+                                        final String abortStreamName, final EventStreamWriter<AbortEvent> abortWriter) {
```

Comment:
In order to initialize stream writers, we need to ensure that their respective streams are first created, The controller service starter asynchronously starts creating these streams while concurrently creating other service objects and starting them, including creation of StreamTransactionMetadataTasks object. Once the streams are created, initializeStreamWriters is invoked, to create stream writers.